src/styles/_catalog.scss

```diff
@@ -1,11 +1,8 @@
 .catalog {
   display: grid;
-  grid-template-columns: repeat(4, $width-card);
   justify-content: center;
+  grid-template-columns: $width-card;
   gap: 48px;
-  flex-wrap: wrap;
-
-  max-width: 944px;
   margin: 0 auto;
 
   @media (min-width: 488px) {
```

Comment:
you can use `@each` to set media with a loop

Reply:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.